prow/github/client.go

```diff
@@ -811,6 +827,14 @@ func (c *client) getUserData() error {
 	// email needs to be publicly accessible via the profile
 	// of the current account. Read below for more info
 	// https://developer.github.com/v3/users/#get-a-single-user
+
+	// record information for the user
+	authHeader := bytes.NewBufferString("Token ")
+	authHeader.Write(c.getToken())
+	hasher := sha256.New()
+	hasher.Write(authHeader.Bytes())
```

Comment:
Why are we not simply using `crypto/sha256.Sum256([]byte(fmt.Sprintf("Token: %s@, c.getToken())))`?

Reply:
Is this python?!

Reply:
I copy-pasted this from how I did it in `ghmetrics.go` -- if the `Sum256` is identical, I can refactor these two to do that. WDYT?

Reply:
yeah please do

Reply:
updated